configure.py

```diff
@@ -284,6 +284,8 @@ def binary(name):
               '/wd4512', '/wd4800', '/wd4702', '/wd4819',
               # Disable warnings about passing "this" during initialization.
               '/wd4355',
+              # Disable warnings about ignored typedef in DbgHelp.h
+              '/wd4091',
```

Comment:
(Mostly note to self: This was reported upstream at https://connect.microsoft.com/VisualStudio/feedbackdetail/view/888527/warnings-on-dbghelp-h )